openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -193,6 +315,16 @@
             charts.push(options);
         }
         charts = sortByTraceOrder(data.trace_order, charts, '_key');
+
+        if (unit == 'adaptive_bytes') {
+            var yRawVal;
+            for( i =0; i<charts.length; i++){
```

Comment:
Please pay attention to the formatting. 


This is something that I would expect the QA checks to take care for me. I will look into QA checks later on.